Upstream: no
Description: In VoidLinux, we always have same version of python in build and host
--- a/configure.ac
+++ b/configure.ac
@@ -9879,8 +9879,6 @@ if test "$build_os" != "cygwin" -a "$ena
     else
         # This allows a lack of system python with no error, we use internal one in that case.
         AM_PATH_PYTHON([3.3],, [:])
-        # Clean PYTHON_VERSION checked below if cross-compiling
-        PYTHON_VERSION=""
         if test "$PYTHON" != ":"; then
             PYTHON_FOR_BUILD=$PYTHON
         fi
